backend/dal/Migrations/v01.10.04/Up/PostUp/01-Agencies.sql

```diff
@@ -44,23 +55,23 @@ DROP TABLE #TRANUsers
 -- Move properties to BCTFA
 UPDATE dbo.[Parcels]
 SET [AgencyId] = @BCTFA
-WHERE [AgencyId] IN (@TRAN, @PLMB, @TIC, @BCT)
+WHERE [AgencyId] IN (@TRAN, @PLMB, @TIC)
```

Comment:
I'm not sure if we need to update existing "TRAN" agencies to the subagency "BCTFA", as I thought Maggie mentioned only properties currently reporting as either parent agency "TRAN" and subagency PLMB or parent agency TRAN and subagency is blank or parent agency is TRAN and subagency is TIC should be updated to subagency BCTFA?

Reply:
I forgot that a property can belong to a subagency and a parent agency using one agency id.